server/middlewares/addDevMiddlewares.js

```diff
@@ -17,7 +17,9 @@ module.exports = function addDevMiddlewares(app, webpackConfig) {
   const middleware = createWebpackMiddleware(compiler, webpackConfig.output.publicPath);
 
   app.use(middleware);
-  app.use(webpackHotMiddleware(compiler));
+  app.use(webpackHotMiddleware(compiler, {
+    heartbeat: 2000,
```

Comment:
heartbeat should be "less than the client's timeout setting - usually set to half its value"
https://github.com/glenjamin/webpack-hot-middleware#middleware

since default timeout is 2s, let's reduce this to 1s **or** increase client timeout to 4s eg by updating query
```
webpack-hot-middleware/client?reload=true&timeout=4000
```

https://github.com/react-boilerplate/react-boilerplate/blob/dev/internals/webpack/webpack.dev.babel.js#L44

Reply:
Yeah, I've tried the `timeout` option already, but somehow the error didn't appear anymore even when I haven't added it to `webpack-hot-middleware/client?reload=true`.
Anyways, I'm ok with set the client default timeout is 2s, and heartbeat is 1s 👍